src/main/java/org/mockito/Mockito.java

```diff
@@ -2702,4 +2696,17 @@ static MockitoDebugger debug() {
     public static MockitoFramework framework() {
         return new DefaultMockitoFramework();
     }
+
+    /**
+     * Allows to manage Mockito strictness (debugging, cleaner tests) without JUnit.
+     * <p>
+     * TODO javadoc
+     *
+     * @param testClassInstance
+     * @param strictness
+     */
+    @Incubating
+    public static MockitoMocking startMocking(Object testClassInstance, Strictness strictness) {
+        return new DefaultMockitoMocking(testClassInstance, strictness, new ConsoleMockitoLogger());
+    }
```

Comment:
Let's do `Mockito.mockitoSession()` for now, but we should take another look at the method creep in this class. It is getting out of hand slowly but steadily :P